contracts/modules/governance/NXMaster.sol

```diff
@@ -20,13 +20,13 @@ import "./external/OwnedUpgradeabilityProxy.sol";
 contract NXMaster is INXMMaster, Governed {
   using SafeMath for uint;
 
-  struct EmergencyPause {
+  struct UnusedStruct0 {
     bool pause;
     uint time;
     bytes4 by;
   }
 
-  EmergencyPause[] public emergencyPaused;
+  UnusedStruct0[] public _unused0;
```

Comment:
You can remove the struct definition and replace all this with just `uint public _unused0;`
The array occupies an entire slot by default.